views/pages/details.html

```diff
@@ -43,6 +43,12 @@
   {{#record.status.courtOrderRevoked}}
     <div class="flag">{{#t}}flags.courtOrderRevoked{{/t}}</div>
   {{/record.status.courtOrderRevoked}}
+
+  {{#record.previousRegistration.systemNumber}}
+    <div class="flag warning">{{#t}}flags.previousRegistration{{/t}}
+      <a id="backToSearchResults" href="/details/{{record.previousRegistration.systemNumber}}?{{querystring}}" class="button">{{record.previousRegistration.date}}</a>
+    </div>
+  {{/record.previousRegistration.systemNumber}}
```

Comment:
I think there should already be one or more flags that appear on re-registrations. I'd imagined this story would just be a case of adding a link to those.

Would that work? (Maybe not with the current version of the API.)

Reply:
We just link to the registration immediately previous to this. When the v1 API is stable we might to something more advanced.